File: PHP/Booking/SubmitOrderResult.php
<?php
namespace LTD_API;

class SubmitOrderResult
{
    private $success;
    private $paymentRedirectUrl;
    
    public function getSuccess()
    {
        return $this->success;
    }
    public function setSuccess($success)
    {
        $this->success = $success;
    }

    public function getPaymentRedirectUrl()
    {
        return $this->paymentRedirectUrl;
    }
    public function setPaymentRedirectUrl($paymentRedirectUrl)
    {
        $this->paymentRedirectUrl = $paymentRedirectUrl;
    }
}

File: Java/Booking/SubmitOrderResult.java
package LTD_API;

public class SubmitOrderResult
{
    private boolean success;
    private String paymentRedirectUrl;
    
    public boolean getSuccess()
    {
        return success;
    }
    public void setSuccess(boolean success)
    {
        this.success = success;
    }

    public String getPaymentRedirectUrl()
    {
        return paymentRedirectUrl;
    }
    public void setPaymentRedirectUrl(String paymentRedirectUrl)
    {
        this.paymentRedirectUrl = paymentRedirectUrl;
    }
}

